tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "allowJs": false,
-    "lib": ["es2017"],
+    "lib": ["es2015", "es2017"],
```

Comment:
I wasn't getting errors with `npm run build` or `npm test` without this, but the TypeScript linter in Atom was complaining about this when declaring a test case with `test('foo', async () => {`

Reply:
Which Atom plugin are you using? I'm not getting any errors in Atom or VSCode without it. I'm using https://github.com/AtomLinter/linter-tslint and https://github.com/atom/language-typescript

Either way I think it's fine to leave it in.